tests/pytests/functional/channel/test_server.py

```diff
@@ -137,7 +137,11 @@ def cb(payload):
         io_loop.stop()
 
     channel.on_recv(cb)
-    server.publish({"tgt_type": "glob", "tgt": [channel_minion_id], "WTF": "SON"})
+    log.info("TEST - RUN PUBLISH")
```

Comment:
```suggestion
    log.info("TEST - RUN PUBLISH")
```
left over? switch log level?